CHANGES

```diff
@@ -228,7 +227,7 @@ Example Codes
   (see examples/primers/parIters.chpl)
 * updated and renamed the chpldoc primer example to reflect chpldoc's changes
   (see examples/primers/chpldoc.doc.chpl)
-* updated the randomnumbers primer to reflect changes to the interface
+* updated the randomNumbers primer to reflect changes to the interface
```

Comment:
In retrospect, I'd probably put a space in between these words (and should have myself) given that I ended up relying on the parenthetical to name the test in the next line...  Makes it seem a bit less redundant...